llrt_utils/src/encoding.rs

```diff
@@ -113,3 +128,13 @@ pub fn bytes_to_hex_string(bytes: &[u8]) -> String {
 pub fn bytes_to_string(bytes: &[u8]) -> String {
     String::from_utf8_lossy(bytes).to_string()
 }
+
+pub fn bytes_to_utf16_string(bytes: &[u8]) -> Result<String, String> {
+    let data16 = bytes
+        .chunks(2)
```

Comment:
There is a nighly “array_chunks” method that can be used. It can be feature gated so it uses that only on nightly (see json_escape for reference on feature gate nightly)

Reply:
I don't know if I have the correct way to turn off `warning`, but I have responded.
Also, I didn't know which type of build this would be used for.

Reply:
This is my first time using feature gate, so I am not confident that I have done it well, but I have responded.